### ISSUE TITLE
1-star review on Google play store for deleted my question so many times. 

### ISSUE THREAD

**Issue by miogpsrocks** [mpv-player/mpv#9999]
1-star review on Google play store for deleted my question so many times. 

You are earning the hate. Fire whoever deleted my question and I will remove my 1-star review. 

I switched back to VLC because you people fail the software can't see any video files and you delete my question twice. 

Enjoy the 1-star review. 

### Log file

Even if you think it's not necessary at first, it might help us later to find
possible issues. Make a log file made with -v -v or --log-file=output.txt, paste
it to https://0x0.st/ or attach it to the github issue, and replace this text
with a link to it.

**Comment by sfan5** [comment]
Spamming volunteers who write the software you use will not get your problem faster.

PS: You should be angry at Google for making file access unreasonably hard with Android 11.

**Comment by miogpsrocks** [comment]
> Spamming volunteers who write the software you use will not get your problem fixed faster.
> 
> PS: You should be angry at Google for making file access unreasonably hard with Android 11.

I am not mad at MPV for all the video disappearing after an update on my Nvidia shield but rather the fact that they make it impossible to alert them to the problem so it can be looked at or fixed. 

They have a place to report a bug everywhere except Android then when I do report the bug, the person could simple move my question to the right category but instead wants to pick a fight with me. 

Its like that 911 operating that got into a fight with the caller and refused to send an ambulance to help save her father. 

This company should thank me for alerting them to the problem. Most apps will post an email that you can send them reports with bugs. MPV is the only company where you have to registered and sign up for an Github account which I did not have. 

I don't know why these forum moderators get this God complex and turn into bullies. I guess they get these tools and they feel like they have to bully everyone and start deleting post TRYING TO HELP YOU!!!!

Bottom line is I had to switch back to VLC and as far as I know MVP is still unusable on the Nvidia shield. 

I used the software fine 1 week ago, this week its worthless.

**Comment by CounterPillow** [comment]
this isn't a forum, this is a bug tracker. We cannot move your issue from one repository administrated by one organisation to the repository of another organisation. Instead of simply opening an issue in the correct repository, you decided to continue opening spammy issues in this repository.

> MPV is the only company

mpv isn't a company.

**Comment by haasn** [comment]
You are delusional. Seek help.

**Comment by ValZapod** [comment]
We do not have a build for android. we do not support android, that is a separate build with android ui, etc, from here: https://github.com/mpv-android/mpv-android from user commented above!

But you should be greatful! Compared to mx player it is compiled for >= 845 snap, okay? So you can compile it for new ARM arch. That is like march native, why macOS M1 macbooks are so fast and why linux on M1 macs is so fast!! Wow. LOL. See build here: https://drive.google.com/file/d/17uK9CVQYSKb0zsmqo-KDaLrJLErwos7e/view?usp=sharing from here https://www.reddit.com/r/AV1/comments/pq669h/custom_mpvandroid_build_with_fully_uptodate_dav1d/

I love it on my galaxy s21 fe, it is so fast! Like wow.

>I genuinely think you are showing signs of some sort of psychosis

I think he does not understand how mpv works.

**Comment by miogpsrocks** [comment]
> You are delusional. Seek help.
> 
> This is not me trying to be mean. I genuinely think you are showing signs of some sort of psychosis.

I hope you have a lot of money because you have a defamation lawsuit coming your way. You better hope you used a VPN when you made that comment.

**Comment by miogpsrocks** [comment]
> We do not have a build for android. we do not support android, that is a separate build with android ui, etc, from here: https://github.com/mpv-android/mpv-android from user commented above!
> 
> But you should be greatful! Compared to mx player it is compiled for >= 845 snap, okay? So you can compile it for new ARM arch. That is like march native, why macOS M1 macbooks are so fast and why linux on M1 macs is so fast!! Wow. LOL. See build here: https://drive.google.com/file/d/17uK9CVQYSKb0zsmqo-KDaLrJLErwos7e/view?usp=sharing from here https://www.reddit.com/r/AV1/comments/pq669h/custom_mpvandroid_build_with_fully_uptodate_dav1d/
> 
> I love it on my galaxy s21 fe, it is so fast! Like wow. Does not work on snap 835 (my tab S4) though, alas.
> 
> > I genuinely think you are showing signs of some sort of psychosis
> 
> I think he does not understand how mpv works.

My complaint is 100% of my video files were missing.  I got people at my house waiting to watch a video, its getting late and no video.  I tried to email support but no email provided. 
Then I find out the only way to contact anyone is to sign up for you group. 

Most andorid or IOS app have a easy way to contact support or to report a bug. I never had to join an entire community on github to inform them of a recent bug which renders the software 100% broken. 

Shield had a work around to get it working for VLC but with MPV, it would not work. 

I did not come on here to trash your software until I was treated like trash for trying to inform you of a very serious problem cause the software not to work at all. 

The mod just had to take my report and pass it along to whoever is in charge. Perhaps say "thanks" optional. 

Instead they wanted to get into a fight. So I follow though on my promise and gave 1-star along with the reason with the bug report. 

It would be nice if you would take my bug report serious instead of instantly deleting it but sadly the only way you will listen is if I provide a 1-star review on android  along with the bug report. 

I did not want to do it but your mods forced my hand. 
For shame

**Comment by haasn** [comment]
> It would be nice if you would take my bug report serious instead of instantly deleting it but sadly the only way you will listen is if I provide a 1-star review on android along with the bug report.

Please, our company cannot survive this... I may even lose my job... 😞 We have to get rid of this 1-star report immediately, it is a disaster for the mpv business model...

.. but unfortunately, our work contracts prevent us from helping unless you go through the proper formality and [submit a bug in the correct project](https://github.com/mpv-android/mpv-android/issues/new/choose)... you have to do this for me, understand.. my life is on the line

**Comment by miogpsrocks** [comment]
https://youtu.be/pJTGAtLH7Hw

Sent from Mail for Windows

**Comment by CounterPillow** [comment]
> > You are delusional. Seek help.
> > This is not me trying to be mean. I genuinely think you are showing signs of some sort of psychosis.
> 
> I hope you have a lot of money because you have a defamation lawsuit coming your way. You better hope you used a VPN when you made that comment.

You're the biggest Karen I've ever seen on GitHub, holy shit. Please ask for the manager next.

**Comment by amariami** [comment]
#9999
oh looks on this number his timing are perfect holy crap...  My belly can't stop laughing.

**Comment by NO-ob** [comment]
checked

**Comment by escapezn** [comment]
classic in classic
